nixos/modules/services/x11/window-managers/fvwm2.nix

```diff
@@ -3,17 +3,17 @@
 with lib;
 
 let
-  cfg = config.services.xserver.windowManager.fvwm;
-  fvwm = pkgs.fvwm.override { enableGestures = cfg.gestures; };
+  cfg = config.services.xserver.windowManager.fvwm2;
+  fvwm2 = pkgs.fvwm2.override { enableGestures = cfg.gestures; };
 in
 
 {
 
   ###### interface
 
   options = {
-    services.xserver.windowManager.fvwm = {
-      enable = mkEnableOption "Fvwm window manager";
+    services.xserver.windowManager.fvwm2 = {
```

Comment:
Does this mean that existing configurations using `services.xserver.windowManager.fvwm` will need to be updated?

Reply:
Ideally yes. However I will make an alias for it.